packages/web-components/fast-components/src/slider/slider.ts

```diff
@@ -50,7 +50,7 @@ export class Slider extends FormAssociated<HTMLInputElement>
     /**
      * The element's value to be included in form submission changed.
      */
-    @attr({ converter: nullableNumberConverter })
+    // @attr({ converter: nullableNumberConverter })
```

Comment:
should we just remove this line?

Reply:
looks like an artifact from removing from slider :)